pkg/forwarder/sync_forwarder.go

```diff
@@ -97,14 +97,17 @@ func (f *SyncForwarder) SubmitSketchSeries(payload Payloads, extra http.Header)
 
 // SubmitHostMetadata will send a host_metadata tag type payload to Datadog backend.
 func (f *SyncForwarder) SubmitHostMetadata(payload Payloads, extra http.Header) error {
-	transactions := f.defaultForwarder.createHTTPTransactions(hostMetadataEndpoint, payload, false, extra)
-	return f.sendHTTPTransactions(transactions)
+	return f.SubmitV1Intake(payload, extra)
```

Comment:
Same as mentioned in the forwarder, this changes the destination endpoint. Intended?